tests/Unit/Shipment/Model/Options/DeliveryOptionsTest.php

```diff
@@ -85,3 +85,30 @@
         ->and($deliveryOptions->isPickup())
         ->toBeTrue();
 });
+
+it('always returns an int as id', function () {
+    $faultyDeliveryOptions = new DeliveryOptions(
+        [
+            'date'         => '2022-02-20 16:00:00',
+            'deliveryType' => '1',
+            'packageType'  => '2',
+        ]
+    );
+
+    $faultyDeliveryOptions2 = new DeliveryOptions(
+        [
+            'date'         => '2022-02-20 16:00:00',
+            'deliveryType' => 3,
+            'packageType'  => 4,
+        ]
+    );
+
+    expect($faultyDeliveryOptions->getDeliveryTypeId())
```

Comment:
wanneer je een rij expects moet chainen moet je nadenken of je beter meerdere tests kan maken en/of `with` kunt gebruiken. Hier zou ik zeggen, maak 2 tests:

```
it('returns package type id as int', ...)->with(...)
```

en


```
it('returns delivery type id as int', ...)->with(...)
```

met dan meerdere cases in je `with` array.